src/app/credExplorer/edgeWeights.js

```diff
@@ -38,15 +36,14 @@ export function byNodeType(
   |}>
 ): EdgeEvaluator {
   return function evaluator(edge: Edge) {
-    const srcDatum = prefixes.find(({prefix}) =>
-      NodeAddress.hasPrefix(edge.src, prefix)
+    const srcDatum = NullUtil.orThrow(
```

Comment:
In this case, I think it was actually cleaner before. I'm fine either way tho.

Reply:
On second thought, I agree: the old version is clearer because it only
performs one test, and it’s staying in a statement context so we don’t
buy anything from using an expression form. I’ll revert this change.
Thanks!